doc/source/parameters/cfd/boundary_conditions_cfd.rst

```diff
@@ -0,0 +1,48 @@
+Boundary conditions - CFD
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subsection's defines the boundary conditions associated with fluid dynamics physics. Lethe supports slip, no-slip, periodic, and dirichlet boundary conditions which are set from arbitrary functions. These functions can be used to define all sort of steady-state and transient velocity boundary conditions such as rotating walls.
```

Comment:
It's me being me again. 
I would't say arbitrary Dirichlet. I would rather stick to slip, no-slip, periodic, velocity inlet,  and outflow.  
Isn't a slip BC a mix between Dirichlet for the normal component of the velocity and Newman for the total stress tensor? Can we define a Dirichlet for pressure now? 
Might be wrong here. 

"Lethe supports slip, no-slip, periodic, outflow, and inlet velocity boundary conditions where the later can be set by arbitrary functions"

Reply:
> It's me being me again. I would't say arbitrary Dirichlet. I would rather stick to slip, no-slip, periodic, velocity inlet, and outflow. Isn't a slip BC a mix between Dirichlet for the normal component of the velocity and Newman for the total stress tensor? Can we define a Dirichlet for pressure now? Might be wrong here.
> 
> "Lethe supports slip, no-slip, periodic, outflow, and inlet velocity boundary conditions where the later can be set by arbitrary functions"

That's a bit tricky about the Dirichlet part. The no-slip boundary condition is just a Function expression with the expression being zero (and this is done for efficiency reasons). You can't really impose outflow with a function. If you do, the simulation is sure to crash. Additionaly, you can use the function to impose a tangential flow, so that is not an inflow, but it's a no-slip with non-zero velocity. If you want to specify an outlet, you would need to specify "nothing" actually. Clearly this can be worded better,  but I'm not sure how.

Reply:
To me outflow would be the pressure set to zero thing. Apparently we interpreted this differently?

Reply:
Yeah outlet was the word. 
It s hard for me to tell cause I have no idea how BCs are imposed in FEM for NS specially how you do it

Reply:
Yeah, outflow/outlet are the no-boundary condition (like zero flux in heat transfer FEM). I'll try to make a better introduction to this section tomorrow.